python/ml4ir/applications/ranking/tests/data/configs/model_config_sanity_tests.yaml

```diff
@@ -0,0 +1,13 @@
+architecture_key: linear
+layers:
+  - type: dense
+    name: linear_layer
+    units: 1
+    kernel_initializer: Ones
```

Comment:
Commented on the document as well. But I don't think just doing this makes the two models the same.
- the optimizer state could be different.
- the order of data mini batches will for sure be different because of the `tf.data` optimization leading to possibly different convergence.
@mohazahran